packages/xdl/src/BundleIdentifier.ts

```diff
@@ -0,0 +1,132 @@
+import { ExpoConfig, getConfigFilePaths, getProjectConfigDescription } from '@expo/config';
```

Comment:
let's add a big doc block here to point to the canonical version of this code in eas-cli, so it's clearly documented that this is a copy/paste fork of it